cells/prelude/library.nix

```diff
@@ -0,0 +1,92 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs.nixpkgs) lib;
+  inherit (inputs.haskellNix.legacyPackages) haskell-nix;
+  inherit (haskell-nix) haskellLib;
+
+  default-compiler = "ghc8107";
+  extra-compilers = ["ghc924"];
+
+  # We clean-up src to avoid rebuild for unrelated changes:
+  src = haskellLib.cleanSourceWith {
```

Comment:
why doesn't `cleanGit` do what you want?

Reply:
yeah removing a few filters that were redundant with flake builtin-in filtering (~ `cleanGit`).